doc/baremetal/openocd-axs.rst

```diff
@@ -140,6 +144,9 @@ For the AXS103 following numbers should appear:
 * 2.0 for the HS34
 * 3.0 for the HS38 (core 0)
 * 4.0 for the HS38 (core 1)
+* 1.0 for the HS47D
```

Comment:
Maybe add mention of the board's FW version, otherwise how come you interpret the same value in 2 different ways?